.github/workflows/pr.yml

```diff
@@ -0,0 +1,13 @@
+name: PR
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  run_pr_commenter:
```

Comment:
Consider adding friendly names to these steps because their names show up in the "checks" box/tab.

Reply:
User-friendly names included in Latest Update:

![image](https://github.com/ni/measurementlink-labview/assets/88401200/53dcd1d9-d501-4a81-b3af-8ac991a0c61f)